lib/rubocop/cop/migration/department_name.rb

```diff
@@ -10,13 +10,18 @@ class DepartmentName < Cop
 
         MSG = 'Department name is missing.'
 
+        DISABLING_COPS_CONTENT_TOKEN = /[A-z,]+/.freeze
```

Comment:
What exactly is the content token? Perhaps some comment would be useful here.

Reply:
Sure! I added a comment about the content token.